app/Http/Controllers/Api/UserController.php

```diff
@@ -26,4 +26,22 @@ public function updateUserData(UpdateUserDateRequest $request, User $user)
 
         return response()->json(['status' => ResponseAlias::HTTP_OK, 'message' => 'Your data has been changed'], ResponseAlias::HTTP_OK);
     }
+
+    public function getAllUsersArray()
+    {
+        $collection = User::pluck('email')->toArray();
+
+        return response()->json(['users' => $collection]);
+    }
+
+    public function getOneAuthUserArray(User $id)
```

Comment:
1) where is the request with policy check? where is the model binding?
2) rename action 'view'